tests/test_photo.py

```diff
@@ -18,111 +18,140 @@
+        self.assertEqual(photo.height, self.photo.height)
+        self.assertEqual(photo.file_size, self.photo.file_size)
 
         self.assertEqual(message.caption, self.caption)
 
     @flaky(3, 1)
     @timeout(10)
-    def test_send_photo_jpg_file(self):
-        message = self._bot.sendPhoto(self._chat_id, self.photo_file)
+    def test_get_and_download_photo(self):
+        new_file = self._bot.getFile(self.photo.file_id)
 
-        thumb, photo = message.photo
+        self.assertEqual(new_file.file_size, self.photo.file_size)
+        self.assertEqual(new_file.file_id, self.photo.file_id)
+        self.assertTrue(new_file.file_path.startswith('https://'))
 
-        self.assertTrue(isinstance(thumb.file_id, str))
-        self.assertNotEqual(thumb.file_id, '')
-        self.assertTrue(isinstance(thumb, telegram.PhotoSize))
-        self.assertEqual(thumb.width, self.thumb['width'])
-        self.assertEqual(thumb.height, self.thumb['height'])
-        self.assertEqual(thumb.file_size, self.thumb['file_size'])
+        new_file.download('telegram.jpg')
+
+        self.assertTrue(os.path.isfile('telegram.jpg'))
 
-        self.assertTrue(isinstance(photo.file_id, str))
-        self.assertNotEqual(photo.file_id, '')
-        self.assertTrue(isinstance(photo, telegram.PhotoSize))
-        self.assertEqual(photo.width, self.width)
-        self.assertEqual(photo.height, self.height)
-        self.assertEqual(photo.file_size, self.file_size)
 
     @flaky(3, 1)
     @timeout(10)
     def test_send_photo_url_jpg_file(self):
-        message = self._bot.sendPhoto(self._chat_id, self.photo_file_url)
+        message = self._bot.sendPhoto(self._chat_id, photo=self.photo_file_url)
 
         thumb, photo = message.photo
 
-        self.assertTrue(isinstance(thumb.file_id, str))
+        self.assertIsInstance(thumb, telegram.PhotoSize)
+        self.assertIsInstance(thumb.file_id, str)
         self.assertNotEqual(thumb.file_id, '')
-        self.assertTrue(isinstance(thumb, telegram.PhotoSize))
-        self.assertEqual(thumb.width, self.thumb['width'])
-        self.assertEqual(thumb.height, self.thumb['height'])
-        self.assertEqual(thumb.file_size, self.thumb['file_size'])
+        self.assertEqual(thumb.width, self.thumb.width)
+        self.assertEqual(thumb.height, self.thumb.height)
+        self.assertEqual(thumb.file_size, self.thumb.file_size)
+
+        self.assertIsInstance(photo, telegram.PhotoSize)
+        self.assertIsInstance(photo.file_id, str)
+        self.assertNotEqual(photo.file_id, '')
+        self.assertEqual(photo.width, self.photo.width)
+        self.assertEqual(photo.height, self.photo.height)
+        self.assertEqual(photo.file_size, self.photo.file_size)
+
+    @flaky(3, 1)
+    @timeout(10)
+    def test_send_photo_url_png_file(self):
+        message = self._bot.sendPhoto(
+            photo='http://dummyimage.com/600x400/000/fff.png&text=telegram', chat_id=self._chat_id)
```

Comment:
Could we maybe use a photo in our github here? Not a fan of depending on external sites (anymore than we already do)